src/components/TxData.js

```diff
@@ -33,6 +33,9 @@ class TxData extends React.Component {
    */
   state = { raw: false, children: false, tokens: [] };
 
+  // Array of token uid that was already found to show the symbol
+  tokensFound = [];
```

Comment:
Do we really need this? Now that the API returns all the tokens and their information, we should be able to just assemble a dict with this info (`Dict[uid, tokenConfig]`) and use it for getting the information. `tokens` on the state wouldn't be needed.

Reply:
Actually, I think `tokens` might still need to be on the state, so we re-render the screen after we get the response from the API. But the rest of the suggestion is still valid.

Reply:
For the explorer the refactor makes total sense. Did it, thanks!